BigBaseV2/src/backend/looped/vehicle/boost_behavior.cpp

```diff
@@ -0,0 +1,27 @@
+#include "backend/looped/looped.hpp"
+#include "core/enums.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	void looped::vehicle_boost_behavior()
+	{
+		auto* const vehicle = g_local_player->m_vehicle;
+
+		if (vehicle && VEHICLE::GET_HAS_ROCKET_BOOST_(self::veh))
+		{
+			if (g->vehicle.boost_behavior == eBoostBehaviors::INSTANT_REFIL && vehicle->m_boost == 0.f) // No Boost Refil Time
+			{
+				vehicle->m_boost = 1.f;
+			}
+			else if (g->vehicle.boost_behavior == eBoostBehaviors::INSTANT_REFIL) // Infinite Boost
```

Comment:
Did you mean to use `eBoostBehaviors::INSTANT_REFIL` here instead of `eBoostBehaviors::INFINITE_BOOST`?

Reply:
Yeah.